packages/lablgtk3-sourceview3/lablgtk3-sourceview3.3.0.beta4/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+
+description: "OCaml interface to GTK+, gtksourceview2 library"
+synopsis: "OCaml interface to GTK+"
+
+maintainer: "garrigue@math.nagoya-u.ac.jp"
+authors: ["Jacques Garrigue et al., Nagoya University"]
+homepage: "http://lablgtk.forge.ocamlcore.org/"
+bug-reports: "https://github.com/garrigue/lablgtk/issues"
+dev-repo: "git+https://github.com/garrigue/lablgtk.git"
+doc: "https://garrigue.github.io/lablgtk/lablgtk-sourceview"
+license: "LGPL with linking exception"
+
+depends: [
+  "ocaml"                {         >= "4.05.0"    }
+  "dune"                 { build & >= "1.4.0"     }
+  "lablgtk3"             {          }
+  # To fix when opam is updated
+  # "lablgtk3"             {         >= "3.0.beta4" }
```

Comment:
@garrigue you want to fix this line , it was temporarily disabled as to workaround a deficiency of `opam pin` which is used in the CI. Once this PR gets merged CI will be able to proceed better.

Ditto for the `gtkspell` one.